.changelog/25562.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/aws_default_vpc_dhcp_options: Fix `missing expected [` error introduced in [v4.20.0](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#4200-june-23-2022)
```

Comment:
```suggestion
resource/aws_default_vpc_dhcp_options: Fix `missing expected` error introduced in [v4.20.0](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#4200-june-23-2022)
```

At first the square bracket confused me. Maybe we drop it?

Reply:
That `[` character is in the error message, so I would prefer to keep it.